SensusService/Probes/User/Scripts/ScriptRunner.cs

```diff
@@ -415,52 +446,37 @@ private void StartRandomTriggerCallbacks()
 #error "Unrecognized platform."
 #endif
 
-                // find next future trigger window, ignoring month, day, and year of windows. the windows are already sorted.
+                // find number of days left in participation horizon
```

Comment:
I might be missing something, but I think you're misinterpreting `ParticipationHorizon`. This is not the end of the study (our studies are currently unbounded). Rather, it is the horizon used to calculate participation. Suppose the horizon is 1 day. Below you'll schedule the callbacks for one day, and that will be it.  How do you keep the scripts going after the initial call to `StartRandomTriggerCallbacks`?